Tahoma crash

Despite the directory Fonts existing there, the verb crashes:
```
[ERROR] org.phoenicis.multithreading.ControlledThreadPoolExecutorService (l.64) - Path "/home/jonasz/.Phoenicis/containers//wineprefix//The Sims 3//drive_c/windows/Fonts" does not exist
	at org.phoenicis.tools.files.FileUtilities.copy(FileUtilities.java:139)
	at <js> cp(Unnamed:56:1251-1284)
	at <js> go(Unnamed:40:1244-1308)
	at <js> install(Unnamed:56:1774-1794)
	at org.graalvm.polyglot.Value.invokeMember(Value.java:459)
	at org.phoenicis.engines.VerbsManager.lambda$installVerb$0(VerbsManager.java:71)
	at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by host exception: java.lang.IllegalArgumentException: Path "/home/jonasz/.Phoenicis/containers//wineprefix//The Sims 3//drive_c/windows/Fonts" does not exist

[WARNING] 
org.graalvm.polyglot.PolyglotException: Path "/home/jonasz/.Phoenicis/containers//wineprefix//The Sims 3//drive_c/windows/Fonts" does not exist
    at org.phoenicis.tools.files.FileUtilities.copy (FileUtilities.java:139)
    at <js>.cp (Unnamed:56)
    at <js>.go (Unnamed:40)
    at <js>.install (Unnamed:56)
    at org.graalvm.polyglot.Value.invokeMember (Value.java:459)
    at org.phoenicis.engines.VerbsManager.lambda$installVerb$0 (VerbsManager.java:71)
    at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval (PhoenicisInteractiveScriptSession.java:35)
    at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1 (BackgroundScriptInterpreter.java:45)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    at java.lang.Thread.run (Thread.java:834)
```

--- a/Engines/Wine/Verbs/Tahoma/script.js
+++ b/Engines/Wine/Verbs/Tahoma/script.js
@@ -35,7 +35,7 @@ class Tahoma {
             .wizard(wizard)
             .archive(tahoma)
             .to(`${prefixDirectory}/drive_c/tahoma/`)
-            .extract(["-L", "-F", "tahoma*.tff"]);
+            .extract(["-L", "-F", "tahoma*.ttf"]);
 
         cp(`${prefixDirectory}/drive_c/tahoma/tahoma.ttf`, fontDirectory);
         cp(`${prefixDirectory}/drive_c/tahoma/tahomabd.ttf`, fontDirectory);

